src/api/v1alpha1/workload.go

```diff
@@ -39,6 +40,8 @@ var AllWorkloads = []string{
 type Workload struct {
 	// For pure Pod, no object reference.
 	corev1.ObjectReference `json:"metadata"`
+	// Replicas of this workload.
+	Replicas appsv1.ReplicaSet `json:"replicas"`
```

Comment:
I see some of these replica set related changes. Why do we need these changes? Is this some gap in Project Narrows that we found?